src/components/collaborative-editing/components/avatars/style.scss

```diff
@@ -20,6 +20,16 @@ $avatarBorderWidth: 2px;
 	border-width: $avatarBorderWidth;
 	border-style: solid;
 
+	// Button reset styles
+	margin: 0;
+	padding: 0;
+	background: transparent;
+	-webkit-appearance: none;
```

Comment:
We can use the non-prefixed `appearance` property and rely on autoprefixer to add the necessary vendor prefixes 👍

Reply:
Ah, nice catch. Copy-pasta 😅